tsconfig.json

```diff
@@ -38,6 +37,9 @@
     "node_modules",
     ".github",
     ".git",
-    ".circleci"
+    ".circleci",
+    // Temporary fix to prevent typescript generated JSX errors in the IDE 
+    // triggered by using baseUrl": "desktop/core/src/desktop/js".
```

Comment:
I spent **a lot** of time trying to figure this one out. The JSX code in the tsx files showed an incorrect error for any components that used interface React.FunctionComponent or other built in JSX stuff like React.Fragment. Removing the baseUrl solved the problem but of course introduced a lot of import errors. The only thing I could find that worked without breaking other typescript files was this solution, but I actually still don't know the root cause of this issue is.

Reply:
Odd. We can look into that later.